.travis/script.sh

```diff
@@ -144,7 +144,7 @@ set -x
 
 [ $BUILD_TARGET != posix-ncp-spi ] || {
     ./bootstrap || die
-    make -f examples/Makefile-posix check configure_OPTIONS="--enable-ftd --enable-ncp=spi --with-examples=posix --with-platform-info=POSIX" || die
+    make -f examples/Makefile-posix check configure_OPTIONS="--enable-ftd --enable-ncp-app=all --enable-cli-app=all --with-ncp-bus=spi --with-examples=posix --with-platform-info=POSIX" || die
```

Comment:
Remove the `--enable-cli-app=all` option.  This build check is not intended to run the python test scripts.